pybamm/models/submodels/transport_efficiency/bruggeman.py

```diff
@@ -1,12 +1,12 @@
 #
-# Class for Bruggemantransport_efficiency
+# Class for Bruggeman transport_efficiency
 #
 import pybamm
 from .base_transport_efficiency import BaseModel
 
 
 class Bruggeman(BaseModel):
-    """Submodel for Bruggeman transport_efficiency
+    """Submodel for transport_efficiency
```

Comment:
Thanks, I like it much better now! Could you add a docstring with the specific model name for each class and, if possible, the reference?